static/js/drafts.js

```diff
@@ -498,21 +499,22 @@ function drafts_scroll(next_focus_draft_row) {
     }
     activate_element(next_focus_draft_row[0].children[0]);
 
+    const scroll_element = ui.get_scroll_element($(".drafts-list"));
+
     // If focused draft is first draft, scroll to the top.
     if ($(".draft-info-box").first()[0].parentElement === next_focus_draft_row[0]) {
-        $(".drafts-list")[0].scrollTop = 0;
+        scroll_element.scrollTop = 0;
```

Comment:
This doesn’t make sense. `scroll_element` is a jQuery object, so `scroll_element.scrollTop` is a [method](https://api.jquery.com/scrolltop/).